.github/workflows/dart.yml

```diff
@@ -0,0 +1,112 @@
+# Created with package:mono_repo v3.1.0-beta
+name: Dart CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  job_001:
+    name: "OS: linux; SDK: dev; PKG: mono_repo; TASKS: `cd ../ && dart mono_repo/bin/mono_repo.dart generate --validate`"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: cedx/setup-dart@v2
+        with:
+          release-channel: dev
+      - uses: actions/checkout@v2
+      - env:
+          PKGS: mono_repo
+          TRAVIS_OS_NAME: linux
+        run: tool/ci.sh command
+  job_002:
+    name: "OS: linux; SDK: dev; PKGS: mono_repo, test_pkg; TASKS: [`dartfmt -n --set-exit-if-changed .`, `dartanalyzer --fatal-infos --fatal-warnings .`]"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: cedx/setup-dart@v2
+        with:
+          release-channel: dev
+      - uses: actions/checkout@v2
+      - env:
+          PKGS: mono_repo test_pkg
+          TRAVIS_OS_NAME: linux
+        run: tool/ci.sh dartfmt dartanalyzer_0
+  job_003:
+    name: "OS: linux; SDK: 2.7.0; PKG: mono_repo; TASKS: `dartanalyzer --fatal-warnings .`"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: cedx/setup-dart@v2
+        with:
+          release-channel: stable
+          version: "2.7.0"
+      - uses: actions/checkout@v2
+      - env:
+          PKGS: mono_repo
+          TRAVIS_OS_NAME: linux
+        run: tool/ci.sh dartanalyzer_1
+  job_004:
+    name: "OS: linux; SDK: 2.7.0; PKG: mono_repo; TASKS: `pub run test -P presubmit --test-randomize-ordering-seed=random`"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: cedx/setup-dart@v2
+        with:
+          release-channel: stable
+          version: "2.7.0"
+      - uses: actions/checkout@v2
+      - env:
+          PKGS: mono_repo
+          TRAVIS_OS_NAME: linux
```

Comment:
Is the reference to travis temporary?

Reply:
Yeah. It's used by the `ci` script to make sure the right thing CLI command is used

Reply:
will we pick a different name in a future update?